package/scripts/mock-data.ts

```diff
@@ -3,12 +3,29 @@ const { Client } = pkg;
 import format from 'pg-format';
 import { faker } from '@faker-js/faker';
 
-function createRandomProduct() {
-  return [
-    faker.commerce.productName(),
-    faker.commerce.productDescription(),
-    faker.commerce.price(10, 20000, 0),
-  ];
+function createRandomProduct(i: number) {
+  const name = faker.commerce.productName();
+  const description = faker.commerce.productDescription();
+  // To test the full-text search we need some products to have
+  // common words in the name and description
```

Comment:
Do we?  Which test relies on this?

Reply:
I added a comment re this.

Reply:
Thanks, I see it. But we don't need such complicated logic here to support that test. We just need to add one `awesome` to one row's description.

Reply:
Haha, yes now I step back I see it's very over-engineered for just that one test. I'll update